ambari-server/src/test/java/org/apache/ambari/server/bootstrap/BootStrapTest.java

```diff
@@ -172,7 +172,7 @@ public void testHostFailure() throws Exception {
     info.setUser("user");
     info.setUserRunAs("root");
     info.setPassword("passwd");
-    BSResponse response = impl.runBootStrap(info);
+    BSResponse response = impl.runBootStrap(info, false);
```

Comment:
Can you please also add test case for validation-type bootstrap?

Reply:
Add two unit tests, one for BootstrapResource.java and another for TestBootstrap.py